pkg/operator/etcdmemberscontroller/etcdmemberscontroller.go

```diff
@@ -138,7 +138,7 @@ func (c *EtcdMembersController) reportEtcdMembers(ctx context.Context, recorder
 		_, _, updateErr := v1helpers.UpdateStatus(ctx, c.operatorClient, v1helpers.UpdateConditionFn(operatorv1.OperatorCondition{
```

Comment:
Slightly suspicious, one line up there's a comment that says "we will never reach here...", but we do see `No quorum` sometimes in Telemetry today.